annotation-file-utilities/tests/system-test/Makefile

```diff
@@ -37,7 +37,13 @@ out2.jaif:
 	CLASSPATH=`pwd`/out1 ${ANNCAT} --class out1/annotations/tests/AnnotationTest.class --out --index out2.jaif
 
 out2.diff: out2.jaif
-	diff -u expected-annos.jaif out2.jaif > out2.diff
+	# Ignore differences between bytecode indices between Java 8 and 9+
+	# TODO: handle different versions?
+	# diff -u expected-annos.jaif out2.jaif > out2.diff
+	grep -v typecast expected-annos.jaif | grep -v instanceof > exp2
```

Comment:
It took me a while to figure out what exp2 and act2 were.  How about using filenames like .xpected-annos-without-instanceof.jaif and out2-without-instanceof.jaif ?  That makes the logic clear.

Reply:
I've changed the names to something more descriptive.